app/models/participant.rb

```diff
@@ -68,6 +68,10 @@ def is_booth_chair?
     !memberships.booth_chairs.blank?
   end
 
+  def booth_chair_organizations
+    memberships.booth_chairs.map(&:organization) & Organization.only_categories(['Fraternity', 'Sorority', 'Independent', 'Blitz', 'Concessions'])
```

Comment:
Reason it is scoped like this? So that SCC / SLICE cannot add themselves to the queues?

Reply:
Can we change the map to use `OrganizationCategory`'s `is_building` field?
We use that everywhere else like in determining wristband colors.

Reply:
Yeah, that's how the sidebar queue add form works too. I'll change it to use is_building.